libraries/ms-helper/src/mcp2515.c

```diff
@@ -387,3 +382,60 @@ StatusCode mcp2515_tx(Mcp2515Storage *storage, uint32_t id, bool extended, uint6
 #endif
   return STATUS_CODE_OK;
 }
+
+StatusCode mcp2515_set_filter(Mcp2515Storage *storage, uint32_t *filters) {
+  // Primarily just copy-pasted from mcp2515_init()
```

Comment:
Is it possible to call this function from `mcp2515_init` instead of copy-pasting?

Reply:
Sort of -- however, `mcp2515_init` does configure a few other things while in config mode that would require the CAN bitrate to be passed into `mcp2515_set_filter`  as well, which would complicate the modules implementing it a bit.  

The simplest way I can think of to get around this is to define a `prv_configure_filters` function that handles everything in `mcp2515_set_filter` from directly after entering config mode to right before we leave it.  Then we can call that in both `mcp2515_init` and `mcp2515_set_filter`, getting rid of most of the copy-pasted code.